src/main/java/athleticli/ui/Parser.java

```diff
@@ -40,7 +40,7 @@ public class Parser {
     private static final String CALORIES_MARKER = "calories";
     private static final String PROTEIN_MARKER = "protein";
     private static final String CARB_MARKER = "carb";
-    private static final String FAT_MARKER = "fat";
+    private static final String FAT_MARKER = "fats";
```

Comment:
One suggestion would be to move the markers to the newly created parameters class.